examples/wkwebview/wkwebview/LinkViewController.swift

```diff
@@ -10,42 +10,44 @@ import WebKit
 
 class LinkViewController: UIViewController, WKNavigationDelegate {
     
-    @IBOutlet var containerView : UIView? = nil
     
     var webView: WKWebView!
     
     override func loadView() {
-        self.webView = WKWebView()
-        self.webView.navigationDelegate = self
-        self.view = webView
+        createWebview()
```

Comment:
I'd like to suggest making `createWebview` more functional by having it return the `WKWebView` that is then assigned to the `view` property and by adding `delegate` parameter to `createWebview`, which is assigned
to the `navigationDelegate` property of the returned `WKWebView`, e.g.:
`view = createWebView(self)`

This sort of code is more explicit and in line with the documentation:
> […] assign the root view of your view hierarchy to the view property.
[UIViewController.loadView()](https://developer.apple.com/reference/uikit/uiviewcontroller/1621454-loadview)

Out of curiosity what is the motivation for the `createWebview()` method?